src/microbe_stage/editor/PatchMapEditorComponent.cs

```diff
@@ -283,4 +314,15 @@ private void MoveToPatchClicked()
     {
         SetPlayerPatch(mapDrawer.SelectedPatch);
     }
+
+    private void OnRevealEntirePatchMapCheatUsed(object sender, EventArgs args)
+    {
+        mapDrawer.IgnoreFogOfWar = true;
+
+        foreach (var patch in mapDrawer.PlayerPatch!.GetAllConnectedPatches())
```

Comment:
Is it safe to assume PlayerPatch is not null here? Wouldn't it be better just to get all patches in the map and mark them as explored?